README.rst

```diff
@@ -6,3 +6,25 @@ SlicerITKUltrasound
 
 A `3D Slicer <http://slicer.org/>`_ extension for ultrasound image formation, processing, and analysis. Interfaces built off the
 `ITKUltrasound <https://github.com/KitwareMedical/ITKUltrasound/>`_ library.
+
+Development
+-----------
+
+To develop SlicerITKUltrasound, first `build 3D Slicer
+<https://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Build_Instructions>`_.
+Version *r25484* (trunk from 2016-10-28) or later is required.
+
+Use CMake to configure SlicerITKUltrasound, and `pass in
+<https://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Build_Module>`_
+the `Slicer_DIR` configuration variable as the
+*<Slicer-Superbuild>/Slicer-build* path. Build SlicerITKUltrasound by running
+`make` or another specific build system generated by CMake.
+
+Run the tests for the extension by executing `ctest` in
```

Comment:
I haven't tested on Windows yet, but it "should work". Adding `-C` suggestion to the docs.